### PR TITLE
Improve time picker focus box-shadow

### PR DESCRIPTION
**Pull request by llorca**
Noticed that time picker still had custom shadows. Making it consistent with already available APIs.

Before:
image

After:
image

### CODE REVIEW

File: packages/datetime/src/_timepicker.scss [hunk 1]
```diff
@@ -93,31 +95,25 @@ $timepicker-control-width: $pt-grid-size * 3.3 !default;
...
     width: $timepicker-control-width;
     height: $timepicker-input-row-inner-height;
     padding: 0;
     text-align: center;
     color: $pt-text-color;
+    transition: $input-transition;
 
     &:focus {
-      box-shadow: border-shadow(1, $blue3, 2px);
+      box-shadow: input-transition-shadow($input-shadow-color-focus, true);
```
Comment (giladgray) [inline packages/datetime/src/_timepicker.scss]: perhaps we should create a mixin for `input-shadows()`?

File: packages/datetime/examples/timePickerExample.tsx [hunk 2]
```diff
@@ -32,7 +32,7 @@ export class TimePickerExample extends BaseExample<ITimePickerExampleState> {
         return [
             [
                 <label className={Classes.LABEL} key="precision">
-                    TimePicker precision
+                    Time picker precision
```
Comment (giladgray) [inline packages/datetime/examples/timePickerExample.tsx]: I think "Precision" should be sufficient, it's clearly for the TimePicker

File: packages/datetime/src/_timepicker.scss (hunk 1 above)
Comment (llorca) [inline packages/datetime/src/_timepicker.scss]: I don't follow. Explain the magic 🎩

Comment (giladgray) [inline packages/datetime/src/_timepicker.scss]: nvm seems this function is the best we can do

**Pull request merged**